### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by abenaTenkorang**
### Awesome Book Page Setup and Functionality

In this pull request, we built a basic website that allows users to add/remove books from a list. This was achieved using JavaScript objects and arrays. We also dynamically modified the DOM and added basic events.

Changes Made

-  A basic HTML form.
-  A container to store user input
-  Use of Local Storage to prevent input data loss
-  Fixing all Linter Errors

### CODE REVIEW

**Comment by abenaTenkorang** [comment]
My partner is not available since yesterday. I have filed a complaint.

File: README.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ We would like to thank all my fellow micronauts.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](https://github.com/abenaTenkorang/Awesome-Books/blob/feature/MIT.md) licensed.
```
Comment (Abenezer-Tilahun) [inline README.md:165]: - [ Optional ] Good job for documenting your README file Professional, Kindly I recommend you to added the section **Future Features**, including 1-3 potential features to be added.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @abenaTenkorang 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (abenaTenkorang) [reply README.md:165]: Future features added. Thank you

**Review by FrederickMih — COMMENTED** [review]
Hi @abenaTenkorang, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly note that you need to create a new branch for any Milestone. This branch has been approved. You are supposed to open a new PR on a new branch for every Milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**